### DISCUSSION TITLE
Break lines in `ui.code`

### DISCUSSION THREAD

**Discussion by cristianmtr** [zauberzeug/nicegui#2899] [Q&A]
### Question

Hello,

Is there a way to configure the `ui.code` element to break long lines, instead of needing to scroll?

**Comment by falkoschindler** [comment]
Hi @cristianmtr,

With some custom CSS you can do that:

```py
ui.add_css('.nicegui-code pre { text-wrap: wrap }')
ui.code('print("Hello, world!")').classes('w-32')
```

Or if you want to target specific `ui.code` elements:
```py
ui.add_css('.code-wrap pre { text-wrap: wrap }')
ui.code('print("Hello, world!")').classes('w-32 code-wrap')
```
